ivadomed/visualize.py

```diff
@@ -171,21 +171,21 @@ def save_img(writer, epoch, dataset_type, input_samples, gt_samples, preds, wand
         elif isinstance(input_samples, list):
             input_samples = input_samples[0]
 
-        grid_img = vutils.make_grid(input_samples,
+        grid_img = vutils.make_grid(torch.flip(input_samples, [2,3]),
```

Comment:
Please add a comment (with reference to issue) to motivate this line of code

Reply:
Great suggestion! Added it

Reply:
@naga-karthik when you address a comment, can you please always point to the commit? Eg:

```
Great suggestion! Added it in <commit number>
```